mobilecoind/src/service.rs

```diff
@@ -5977,4 +5988,29 @@ mod test {
             .expect("Failed getting processed block");
         assert_eq!(response.get_tx_outs().len(), 1);
     }
+
+    #[test_with_logger]
+    fn test_get_release_version(logger: Logger) {
+        let mut rng: StdRng = SeedableRng::from_seed([23u8; 32]);
+        let sender = AccountKey::random(&mut rng);
+
+        // 1 known recipient, 3 random recipients and no monitors.
+        let (_ledger_db, _mobilecoind_db, client, _server, _server_conn_manager) =
+            get_testing_environment(
+                BLOCK_VERSION,
+                3,
+                &[sender.default_subaddress()],
+                &[],
+                logger.clone(),
+                &mut rng,
+            );
+
+        let expected_version = String("1.3.0-pre0");
+
+        // Send request.
+        let response = client
+            .get_release_version()
+            .expect("Incorrect version returned");
```

Comment:
Builds off @remoun's suggestion of not hardcoding the version string. 
It feels a bit like it's testing a re-implementation of the code.
It just happens that `CARGO_PKG_VERSOIN` is the common way to get the version in code and one is testing that that is the value being returned.
```suggestion
        let expected_version = env!("CARGO_PKG_VERSION");

        // Send request.
        let response = client
            .get_release_version(&mc_util_grpc::Empty::new())
            .expect("Failed to get version");
```

Reply:
Yea! That was my thoughts initially, so that's why I hard-coded the version string :/

Reply:
This seems OK. I would also verify that the string is not empty.